### PR TITLE
reset user and post entries before each cypress test

### PR DESCRIPTION
**Pull request by ddrmv**
Co-authored-by: oahciy <yq2550@gmail.com>

### CODE REVIEW

**Comment by ddrmv** [comment]
[(M) Cypress clear database?](https://trello.com/c/W7gYBXMJ/50-m-cypress-clear-database)

File: cypress/plugins/index.js [hunk 1]
```diff
@@ -11,7 +11,28 @@
...
+      mongoose.connect("mongodb://0.0.0.0/acebook_test");
+      const result = mongoose.connection.collection("posts").deleteMany({});
+      return result;
+    },
+  });
+
+  on("task", {
+    async emptyUsers() {
+      mongoose.connect("mongodb://0.0.0.0/acebook_test");
+      const result = mongoose.connection.collection("users").deleteMany({});
```
Comment (ritaaktay) [inline cypress/plugins/index.js:34]: If this is async, do we have to `await` the `deleteMany({})` ?

File: cypress/plugins/index.js [hunk 2]
```diff
@@ -11,7 +11,28 @@
...
+// module.exports = function () {
+//   // `on` is used to hook into various events Cypress emits
+//   // `config` is the resolved Cypress config
+// };
+
+module.exports = async (on) => {
+  on("task", {
+    async emptyPosts() {
+      mongoose.connect("mongodb://0.0.0.0/acebook_test");
+      const result = mongoose.connection.collection("posts").deleteMany({});
```
Comment (ritaaktay) [inline cypress/plugins/index.js:26]: `await` ?

File: cypress/plugins/index.js [hunk 3]
```diff
@@ -11,7 +11,28 @@
...
+  });
+
+  on("task", {
+    async emptyUsers() {
+      mongoose.connect("mongodb://0.0.0.0/acebook_test");
+      const result = mongoose.connection.collection("users").deleteMany({});
+      return result;
+    },
+  });
+};
```
Comment (ritaaktay) [inline cypress/plugins/index.js:38]: We now also have comments so add:
`module.exports = async (on) => {
  on("task", {
    async emptyPosts() {
      mongoose.connect("mongodb://0.0.0.0/acebook_test");
      const result = mongoose.connection.collection("comments").deleteMany({});
       return result;
    },
  });
};`

File: cypress/plugins/index.js (hunk 2 above)
Comment (44jovi) [reply cypress/plugins/index.js:26]: test without `async` first

**Pull request merged**